azkaban-common/src/test/java/azkaban/executor/ExecutionFlowDaoTest.java

```diff
@@ -357,4 +357,12 @@ private void assertTwoFlowSame(final ExecutableFlow flow1, final ExecutableFlow
     assertThat(new HashSet<>(flow1.getEndNodes())).isEqualTo(new HashSet<>(flow2.getEndNodes()));
   }
 
+  /**
+   * restores the clock; see {@link #testFetchEmptyRecentlyFinishedFlows()}
+   */
+  @After
+  public void clockReset() {
+    DateTimeUtils.setCurrentMillisOffset(0);
```

Comment:
In the class `DefaultMailCreatorTest`,

https://github.com/azkaban/azkaban/blob/c75ea9beaea61ef3d92a2948b0d16834e078fb1f/azkaban-common/src/test/java/azkaban/executor/mail/DefaultMailCreatorTest.java#L53

Should we also change DataTimeUtils back when that test finishes?

Reply:
No need to change it, it's already done properly:
https://github.com/azkaban/azkaban/blob/c75ea9beaea61ef3d92a2948b0d16834e078fb1f/azkaban-common/src/test/java/azkaban/executor/mail/DefaultMailCreatorTest.java#L84

Reply:
Thanks for checking!